src/main/kotlin/fp/Lambda.kt

```diff
@@ -0,0 +1,19 @@
+package fp
+
+class Lambda {
+    private fun sumIf(numbers: List<Int>, predicate : (Int)->Boolean) : Int {
+        return numbers.filter { predicate(it) }.sum()
```

Comment:
함수 레퍼런스를 넘겨줄 수도 있습니다.
```suggestion
        return numbers.filter(predicate).sum()
```